R/latex_dependencies.R

```diff
@@ -60,3 +60,21 @@ validate_latex_dependency <- function(list) {
 has_latex_dependencies <- function(knit_meta) {
   has_dependencies(knit_meta, "latex_dependency")
 }
+
+# Convert a yaml structured input to a list of latex dependencies
+yaml_to_latex_dependencies <- function(x){
+  x_yaml <- try(yaml_load_utf8(x))
+  if(class(x_yaml) == "try-error"){
```

Comment:
Just for future records, this is the wrong way to check classes. The correct way is `inherits(x, class)`, because `class(x)` can be a vector.